app/e2e/outgoingCall.test.js

```diff
@@ -0,0 +1,149 @@
+    await expect(element(by.id('client_text_input'))).toHaveText('Client-Web');
+  });
+
+  if (device.getPlatform() === 'android') {
+    // Note: Call to 3156703541 has TwiML response length of 10s
+    describe('Android:', () => {
+      describe('Connect', () => {
+        it('should make a successful PTSN call', async () => {
+          await element(by.id('dialpad_button_3')).tap();
+          await element(by.id('dialpad_button_1')).tap();
+          await element(by.id('dialpad_button_5')).tap();
+          await element(by.id('dialpad_button_6')).tap();
+          await element(by.id('dialpad_button_7')).tap();
+          await element(by.id('dialpad_button_0')).tap();
+          await element(by.id('dialpad_button_3')).tap();
+          await element(by.id('dialpad_button_5')).tap();
+          await element(by.id('dialpad_button_4')).tap();
+          await element(by.id('dialpad_button_1')).tap();
+          await element(by.id('call_button')).tap();
+          await waitFor(element(by.id('active_call'))).toBeVisible();
+          await waitFor(element(by.id('call_status'))).toHaveText('ringing');
+          await waitFor(element(by.id('call_status'))).toHaveText('00:05');
+          await waitFor(element(by.id('active_call'))).not.toBeVisible();
+        });
+
+        it('should validate call is disconnected after callee disconnects the call', async () => {
+          await element(by.id('dialpad_button_3')).tap();
+          await element(by.id('dialpad_button_1')).tap();
+          await element(by.id('dialpad_button_5')).tap();
+          await element(by.id('dialpad_button_6')).tap();
+          await element(by.id('dialpad_button_7')).tap();
+          await element(by.id('dialpad_button_0')).tap();
+          await element(by.id('dialpad_button_3')).tap();
+          await element(by.id('dialpad_button_5')).tap();
+          await element(by.id('dialpad_button_4')).tap();
+          await element(by.id('dialpad_button_1')).tap();
+          await element(by.id('call_button')).tap();
+          await waitFor(element(by.id('active_call'))).toBeFocused();
+          await waitFor(element(by.id('active_call')))
+            .not.toBeFocused()
+            .withTimeout(12000);
+        });
+
+        it('should allow user to disconnect the call', async () => {
+          await element(by.id('dialpad_button_3')).tap();
+          await element(by.id('dialpad_button_1')).tap();
+          await element(by.id('dialpad_button_5')).tap();
+          await element(by.id('dialpad_button_6')).tap();
+          await element(by.id('dialpad_button_7')).tap();
+          await element(by.id('dialpad_button_0')).tap();
+          await element(by.id('dialpad_button_3')).tap();
+          await element(by.id('dialpad_button_5')).tap();
+          await element(by.id('dialpad_button_4')).tap();
+          await element(by.id('dialpad_button_1')).tap();
+          await device.disableSynchronization();
```

Comment:
What does this do?

Reply:
Detox tries to optimize by "synchronizing" https://wix.github.io/Detox/docs/troubleshooting/synchronization/ . But I think there is some kind of bug triggered by their optimization when trying to click the "end call" button. When in the normal "synchronizing" mode detox cannot find the "end call" button, even though I see it as detox steps through the test case. I found a stackoverflow where someone recommended disabling this "testing opimization" https://stackoverflow.com/questions/59412749/detox-detect-that-element-was-displayed